### DISCUSSION TITLE
Monitoring bodies in different countries

### DISCUSSION THREAD

**Discussion by sander-nl** [Nordic-Accessibility-Community-Group/en-301-549-resources-and-eaa-monitoring#3] [General]
This thread is meant to provide information about the monitoring bodies in different (EU) countries: which organisation(s) monitor compliance to accessibility laws, what do they do and how?

**Comment by erikgustafsson** [comment]
### Sweden
#### Public sector (Web directive)
[Digg](https://www.digg.se/)

#### Private sector (EAA)
[Swedish Post and telecommunications agency, PTS (Post- och Telestyrelsen, PTS) ](https://pts.se/digital-inkludering/lagen-om-vissa-produkters-och-tjansters-tillganglighet/) that monitors electronic communication services, bank and e-commerce. PTS is also the main agency that coordinates the other monitoring agencies:

- [Mediemyndigheten](https://mediemyndigheten.se/ansokan-och-registrering/tillganglighet-i-medier/), for services that provides audiovisual media.
- [Konsumentverket](https://www.konsumentverket.se/) and [Transportstyrelsen](https://www.transportstyrelsen.se/sv/om-oss/det-har-ar-transportstyrelsen/vart-uppdrag-och-arbetssatt/hallbar-utveckling/tillganglighet-i-transportsystemet/), for transportation services.
- [Myndigheten för tillgängliga medier (MTM)](https://www.mtm.se/lagkrav-pa-e-bocker/) for E-books.

**Comment by sander-nl** [comment]
### The Netherlands

#### Public sector (WAD)
Logius, via [Digitoegankelijk](https://www.digitoegankelijk.nl/).

#### Products and services (EAA)
- [Authority for Consumers and Markets](https://www.acm.nl/en/authority-consumers-and-markets) for e-commerce and electronic communication services.
- [Authority for the Financial Markets](https://www.afm.nl/en/sector) for banking services.
- [Dutch Media Authority](https://www.cvdm.nl/english-summary-dutch-media-authority/) for e-books and audiovisual services.
- [Dutch Authority for Digital Infrastructure](https://www.rdi.nl/radiocommunications-agency) for products, including e-readers. 
- [Human Environment and Transport Inspectory](https://english.ilent.nl/) for transportation services.
- [Inspectorate of Justice and Security](https://www.inspectie-jenv.nl/english) for emergency number 112.

_Source: https://www.acm.nl/nl/toegankelijkheid/toegankelijkheid-van-websites-en-apps_

**Comment by 2biazdk** [comment]
I've reached out to the Danish [Digitaliseringsstyrelsen](https://en.digst.dk/) (the Danish Agency for Digital Government) and [Erhvervsstyrelsen](https://danishbusinessauthority.dk/) (the Danish Business Authority) to clarify how they plan to monitor, enforce, and implement the EAA. I'll share an update here as soon as I hear back.

**Comment by 2biazdk** [comment]
Here’s what I’ve learned over the past weeks about how **Denmark** plans to monitor the implementation of the European Accessibility Act:

Denmark enforces the EAA through the "Law on Accessibility of Products and Services" ("Tilgængelighedsloven"), with different authorities responsible for various sectors:

- Sikkerhedsstyrelsen (Danish Safety Technology Authority): The primary enforcement body
  - E.g., all products, banking services (on behalf of the Danish Financial Supervision Authority), e-books, e-commerce (on behalf of the Danish Competition and Consumer Authority), and services that provide access to audiovisual media (on behalf of the Danish Agency for Digital Government)
- Energistyrelsen (Danish Energy Agency): Responsible for electronic communication services
  - E.g., Internet providers, mobile network services
- Trafikstyrelsen (Danish Transport Authority): Covers accessibility in transport
  - E.g., airlines, bus and train companies, their websites, apps, and digital ticketing systems
- Søfartsstyrelsen (Danish Maritime Authority): Oversees accessibility in ship transport
  - Websites, mobile apps, and e-ticketing for passenger ferries and cruise lines
- Finanstilsynet (Danish Financial Supervisory Authority): Ensures accessibility in banking services
  - E.g., websites and apps for banks, investment platforms, and digital payment services

---

## New Information from Erhvervsministeriet (March 4, 2025)

- Erhvervsstyrelsen (Danish Business Authority) does not have an enforcement role. Instead, responsibilities are split between the Erhvervsministeriet, Justitsministeriet, Digitaliseringsministeriet, and Transportministeriet.
- Sikkerhedsstyrelsen has begun an industry outreach initiative to provide targeted guidance to affected businesses and organizations.
- For further clarification or enforcement-related questions, Sikkerhedsstyrelsen should be contacted directly at sik@sik.dk.
- More details can be found in their official guidance portal: [Tilgængelighed for e-handelstjenester | Sikkerhedsstyrelsen](https://www.sik.dk/erhverv/produkter/tilgaengelighed-produkter-og-tjenester/tilgaengelighed-e-handelstjenester).

---

What remains unclear is how enforcement will work in practical terms—for instance, how violations will be handled, and whether specific penalties, fines, or compliance check mechanisms are in place.

For more details (in Danish):
- Sikkerhedsstyrelsen's overview and FAQ: https://www.sik.dk/erhverv/produkter/tilgaengelighed-produkter-og-tjenester/tilgaengelighed-e-handelstjenester
- Sikkerhedsstyrelsen’s general guidance: https://www.sik.dk/erhverv/produkter/generel-produktsikkerhed/love-og-regler/der-kommer-nye-krav-om-tilgaengelighed-produkter-og-tjenester
- Here's a useful breakdown of monitoring bodies from Useit Consulting by Stein-Erik Skotkjerra: https://useit-consulting.dk/aktuelt/hvem-forer-kontrol-med-hvad-i-tilgaengelighedsloven/

**Comment by erikgustafsson** [comment]
Added a list of countries [monitoring agencies and accessibility requirements](https://github.com/Nordic-Accessibility-Community-Group/working-with-EN-301-549/blob/main/monitoring-agencies-information.md).

**Comment by 2biazdk** [comment]
FYI - I've begun maintaining this comprehensive guide (in Danish) about how the EAA comes to life in Denmark: https://forlaens.com/digital-tilgaengelighed-i-danmark-en-praktisk-vejledning/

**Reply by erikgustafsson** [reply]
That is amazing!

**Comment by cmuncharaz** [comment]
The body coordinating all the local market surveillance authorities in Spain was created on 25 February 2026. PR #38 updates this information in the table of countries under the EAA. Who should review the PR, @erikgustafsson?
